vendor/dmn-js-properties-panel.js

```diff
@@ -0,0 +1,3 @@
+const { returnOrThrow } = require('../helper');
```

Comment:
@barmac Could you explain what the idea of this is? Making sure the version of the properties panel is appropriate? And if so, would a BPMN equivalent make sense as well?

Reply:
[We do have an equivalent for the bpmn-js-properties-panel](https://github.com/camunda/camunda-modeler-plugin-helpers/blob/master/vendor/bpmn-js-properties-panel.js#L3).

The idea is that the plugin helpers throw an error with a meaningful message (`Not compatible with Camunda Modeler < 5.0.0`) instead of complaining about property of undefined.

Reply:
Okay cool thanks for the insight. In that case LGTM 👍